.github/workflows/main.yaml

```diff
@@ -4,9 +4,7 @@ on:
   push:
     branches: "*"
   pull_request:
-    branches: master
-  schedule:
-    - cron: "0 0 * * *"
```

Comment:
I've disabled the cron job here since we haven't been following it closely. We can bring this back if we find it to be an issue.